vulnerabilityAlerts.json

```diff
@@ -0,0 +1,7 @@
+{
+    "vulnerabilityAlerts": {
+        "labels": [
+            "security"
+        ]
+    }
```

Comment:
This setting is exemplified in the document.

<img width="795" alt="image" src="https://github.com/hatena/renovate-config/assets/909674/1323184e-3bd6-4930-9254-da321d97b623">